### DISCUSSION TITLE
how is it calculated in the centerfreq

### DISCUSSION THREAD

**Discussion by cjamcu** [rtl-airband/RTLSDR-Airband#350] [Q&A] [answered]
Hello, how is it calculated in the centerfreq. I have 4 frequencies and there is one that cannot be heard, I have realized that if I play with that number I can get one or the other to be heard. In scan mode if I can listen to all 4 at the same time without problems. The frequencies are 118.0, 121.1, 121.9, 123.7. If I put centerfreq 122.4 I hear 121.1 , 121.9 and 123.7. If I put centerfreq 121 I hear only 121.1,121.9. I have not been able to listen to the frequency 118.0 in multichannel mode.

Thank you

**Comment by feelingwalnut** [comment, marked as answer]
Take your device's useable bandwidth, divide it by two.
Add that to your lowest frequency, that is your center frequency. Adding the other half of your bandwidth should give you your highest frequency.

2mhz useable bandwidth set at 100mhz center freq. gives you coverage for 99mhz to 101mhz
Rtlsdr's sometimes have dead spots and aren't so strong on the edges of their 2.3?MHz bandwidth. This needs to be factored and sometimes moving 0.1mhz up or down affects reception.

It looks like your example is over a 5mhz spread, is your device able to cover that?
